tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -274,6 +271,22 @@ void InternalCommandsHandler::addBlock(VersionedUpdates &verUpdates, BlockMerkle
   const auto newBlockId = m_blockAdder->add(std::move(updates));
   ConcordAssert(newBlockId == currBlock + 1);
 }
+
+bool InternalCommandsHandler::checkKeyInBlockAccumulatedRequests(
+    const std::string &key,
+    VersionedUpdates &blockAccumulatedVerUpdates,
+    BlockMerkleUpdates &blockAccumulatedMerkleUpdates) const {
+  if (auto it = blockAccumulatedVerUpdates.getData().kv.find(key);
+      it != blockAccumulatedVerUpdates.getData().kv.end()) {
```

Comment:
It's the first time I see such a syntax :-). Please take 'auto it = blockAccumulatedVerUpdates.getData().kv.find(key);' outside the 'if' for a better readability.

Reply:
Please find updated code with changes